web/packages/teleport/src/Assist/service.ts

```diff
@@ -101,6 +101,8 @@ export async function resolveServerMessage(
   }
 }
 
+// TODO(zmb3): check with Ryan about replacing this with streaming
```

Comment:
cc @ryanclark how critical is `getSessionEvents` / `getSessionStream` given the de-investment in assist?

I'd really like to remove these APIs from the frontend for v15 so that I can remove them from the backend in v16.